docs/src/components/DocSearch.css

```diff
@@ -1,3 +1,11 @@
+:root {
+  /* primary colors */
+  --gestalt-colorGray0: #fff;
```

Comment:
@AlbertCarreras please don't prefix these with gestalt - we should only do that for ones that we export in our library.

Reply:
![image](https://user-images.githubusercontent.com/10593890/87371890-2607a900-c53b-11ea-80ce-b0abe6313bbb.png)

Reply:
@AlbertCarreras yes - we need to disable that rule in the `docs/` directory.

Reply:
@cvuerings 

The way dark mode is implemented depends on the naming of the colors variables. 

![image](https://user-images.githubusercontent.com/10593890/87467475-dec9f880-c5cc-11ea-98c7-da51814e6d59.png)